app/services/email_sender/notify_sender.rb

```diff
@@ -0,0 +1,37 @@
+require "notifications/client"
+
+module EmailSender
+  class NotifySender
+    def call(options)
+      unpack(options)
+      raise ArgumentError, "address is required" if @address.nil?
+      send
+    end
+
+  private
+    attr_accessor :address, :template_id
+
+    def unpack(options)
+      raise ArgumentError, "call expects a hash" unless options.is_a?(Hash)
+      @address = options[:address]
```

Comment:
I'm not sure I like the idea of storing the current state (`address` and `template_id`) of the sender as instance variables instead of just using local variables and passing arguments. It doesn't sit well with me, and it presumably means this class isn't thread safe (I don't know if this matters or not in Rails, but it seems unnecessary to have that restriction on such a simple class).